GitUI/CommandsDialogs/FormMergeBranch.cs

```diff
@@ -144,7 +143,8 @@ private void NonDefaultMergeStrategy_CheckedChanged(object sender, EventArgs e)
 
         private void strategyHelp_LinkClicked(object sender, LinkLabelLinkClickedEventArgs e)
         {
-            strategyToolTip.SetToolTip(strategyHelp, _strategyTooltipText.Text);
+            OsShellUtil.OpenUrlInDefaultBrowser(
+                UserManual.UserManual.UrlFor("branches", "advanced-merge-options"));
```

Comment:
This doesn't look pretty... Please use `using`.

Reply:
@RussKie I tried the following but was met with a compilation error:

![image](https://user-images.githubusercontent.com/14078392/198264808-a7a78a5f-f509-413f-9b9c-4422e692900e.png)

![image](https://user-images.githubusercontent.com/14078392/198264904-905a2cf9-defb-420d-b427-de3b41ca1703.png)


My only guess is that since the namespace name is the same as the class name, the compiler is confused, thinking I am intending the namespace name instead.

Any suggestions here or just leave it alone?

Reply:
No worries then.